sympy/functions/special/bessel.py

```diff
@@ -1495,6 +1495,9 @@ def _eval_rewrite_as_hyper(self, z, **kwargs):
         pf2 = z / (root(3, 3)*gamma(Rational(1, 3)))
         return pf1 * hyper([], [Rational(2, 3)], z**3/9) - pf2 * hyper([], [Rational(4, 3)], z**3/9)
 
+    def _eval_rewrite_as_tractable(self, z, limitvar=None, **kwargs):
+        return sqrt(pi)*z**(1/4)*exp(-2*z**(3/2)/3)/2*_airyais(z)
```

Comment:
This is using float like `1/4`. Is this tested?

Reply:
Yeah I think we should be using `S(1)/4` or `S(3)/2` at all such instances !